pyproject.toml

```diff
@@ -23,6 +23,7 @@ httpx = "^0.23.0"
 storage3 = "^0.5.2"
 supafunc = "^0.2.2"
 python-semantic-release = "7.34.6"
+faker = "^18.10.1"
```

Comment:
I think faker ought to be a dev dependency

Reply:
yes, you are right @anand2312 the faker should be at dev dependency.